server/controllers/userChapterController.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response } from 'express';
+import { Chapter, User } from 'server/models';
+import { UserChapter } from 'server/models/UserChapter';
+
+// The whole model is a json response, fix that if there's some sensitive data here
+
+export default {
+  async create(req: Request, res: Response) {
```

Comment:
This is more of a `join` action, since here a user joins a chapter (event though we create a new userChapter object)